system-test/src/main/java/org/apache/mesos/elasticsearch/systemtest/SeedDataContainer.java

```diff
@@ -32,4 +32,9 @@ protected CreateContainerCmd dockerCommand() {
                 .withName("seed_data_" + new SecureRandom().nextInt());
     }
 
+    @Override
+    public String getRole() {
+        return "seed_data";
```

Comment:
Are you sure this will work? I haven't seen a definition of the role "seed_data" anywhere?